### PR TITLE
#447 Add Ant design and implement layout system for client view

### PR DESCRIPTION
**Pull request by stephanevieira75**

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/app.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import '~components/constants.scss';
+@import '~antd/dist/antd.less';
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/app.less:2]: I don't want to use less except for main Antd theme. Can we keep scss file extension ?

File: twake/frontend/src/app/scenes/Client/WorkspacesBar/WorkspacesBar.js [hunk 2]
```diff
@@ -40,7 +41,7 @@ export default class WorkspacesBar extends Component {
     }
 
     return (
-      <div className="workspaces_view fade_in">
+      <Layout.Sider width="60px" className="workspaces_view fade_in">
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/WorkspacesBar/WorkspacesBar.js:44]: Can we put the 60px value in the css class workspaces_view ?

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Just small changes to do,
very good work !

File: twake/frontend/src/app/scenes/Client/app.less (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/app.less]: Done

File: twake/frontend/src/app/scenes/Client/WorkspacesBar/WorkspacesBar.js (hunk 2 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/WorkspacesBar/WorkspacesBar.js]: We need to keep the width prop or antd design will add a default style :
Capture d’écran de 2020-10-22 17-59-12

**Comment by stephanevieira75** [comment]
> Just small changes to do,
> very good work !

All good for me :smile:

**Pull request merged**